portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/IndividualsAttributeBreakdown.testcase

```diff
@@ -144,8 +144,12 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8658 | Test Summary: Delete a new breakdown"
+	@ignore = "true"
 	@priority = "5"
 	test DeleteNewBreakdown {
+
+		// CSV data source tests are no longer possible to run due to this ticket LRAC-11651
+
```

Comment:
Hey @manuferreira, This test can be done using another breakdown attribute that does not need the CSV (Ex: givenName, email and etc)

I think you can undo this change and create a ticket for us to refactor this test or you can do it yourself in your ticket